### PR TITLE
[Task] Description about how to find extensions and renaming management

### PR DESCRIPTION
**Pull request by mkiebele**

### CODE REVIEW

File: Documentation/Extensions/HowToFindExtensions.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+How to find extensions
+======================
+
+For beginners, finding the right extensions can enhance their TYPO3 experience
+significantly. Here is a brief guide on how to search for extensions and where
+to find the necessary information.
+
+#.  `TYPO3 Extension Repository (TER) <https://extensions.typo3.org/>`__
+
+    The TYPO3 Extension Repository (TER) is online platform, that hosts
```
Comment (brotkrueml) [inline Documentation/Extensions/HowToFindExtensions.rst:15]:
```suggestion
    The TYPO3 Extension Repository (TER) is an online platform, that hosts
```

File: Documentation/Extensions/HowToFindExtensions.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    *   Filter results
+
+#.  `Packagist <https://packagist.org//>`__
+
+    Packagist serves as the default package repository for PHP packages using Composer,
+    including TYPO3 extensions. You can search for TYPO3 extensions on Packagist
+    by entering relevant keywords. Many TYPO3 extensions are available via Composer,
+    which allows for easier dependency management and installation. This is
+    particularly useful for developers looking to integrate extensions into
+    their projects.
```
Comment (brotkrueml) [inline Documentation/Extensions/HowToFindExtensions.rst:30]: You can also filter by TYPO3 extensions:

image

https://packagist.org/?type=typo3-cms-extension

File: Documentation/Extensions/HowToFindExtensions.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+
+..  _tips-for-choosing-extensions:
+
+Tips for Choosing the right TYPO3 Extension
```
Comment (brotkrueml) [inline Documentation/Extensions/HowToFindExtensions.rst:35]:
```suggestion
Tips for choosing the right TYPO3 extension
```

We use title case in the docs.

File: Documentation/Extensions/HowToFindExtensions.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+*   Performance and security:
+
+    Extensions can affect your site’s performance, so choose extensions that
+    are optimized and well-coded. Additionally, check for any reported security
+    vulnerabilities, particularly for older extensions, to keep your site secure.
+
+*   Customizability and flexibility:
+
+    If you have specific needs, ensure that the extension is flexible enough to
+    be customized or configured as required. Some extensions provide hooks or
```
Comment (brotkrueml) [inline Documentation/Extensions/HowToFindExtensions.rst:74]:
```suggestion
    be customized or configured as required. Some extensions provide PSR-14 events or
```

As hooks are a dying species, we should avoid them in new docs.

File: Documentation/Extensions/InstallingExtensions.rst [hunk 5]
```diff
@@ -1,20 +1,17 @@
 .. include:: /Includes.rst.txt
 
-.. _extensions_management:
+.. _installing-extensions:
 
-===================
-Managing Extensions
-===================
+=====================
+Installing Extensions
```
Comment (brotkrueml) [inline Documentation/Extensions/InstallingExtensions.rst:6]:
```suggestion
Installing extensions
```

**Review by linawolf — APPROVED** [review]

**Pull request merged**